test/test.js

```diff
@@ -7,9 +7,12 @@ var tcpp = require('tcp-ping');
 var outputTangerine = fs.readFileSync(__dirname + "/tangerine.txt", {
 	encoding: "utf8"
 });
+var fixture1Css = fs.readFileSync(__dirname + "/fixture-1/style.css", {
+	encoding: "utf8"
+});
 
-var testEqual = function(input, output, opts, done) {
-	postcss([plugin(opts)]).process(input).then(function(result) {
+var testEqual = function(input, output, opts, done, postcssOptions) {
```

Comment:
By node convention, callback parameter (done) must be last parameter

Reply:
Fixed.

Would you like me to bump the version number? If so, should I bump it to 2.3.0?